Fix typo in providers/tcp.js

emilecantin opened the pull request:
caused a crash in some cases.

tkurki commented:
Ok, this one is really weird or I have typoed something: this was broken with supposedly automatic reformat with prettier-standard.

Did you notice that I introduced prettier-standard? If you do `npm install` it should install git precommit hook that formats changed files and this change would come in without the semicolon.

emilecantin commented:
I just fixed it via the github web interface, so I didn't notice the precommit hook. Fixed the semicolon (I'm of the other school of thought that always puts the semicolons explicitely, but your project, your rules!)
